README.md

```diff
@@ -126,16 +124,16 @@ Start by [adding widgets](https://www.algolia.com/doc/guides/building-search-ui/
 
 We support the **last two versions of major browsers** (Chrome, Edge, Firefox, Safari).
 
-To support [IE11](https://en.wikipedia.org/wiki/Internet_Explorer_11), we recommend loading [polyfill.io](https://polyfill.io) with the following URL `https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.includes`
+Please refer to the [browser support](https://www.algolia.com/doc/guides/building-search-ui/installation/js/#browser-support) section in the documentation to use InstantSearch.js on other browsers.
 
 ## Contributing
 
 We welcome all contributors, from casual to regular 💙
 
-* **Bug report**. Is something not working as expected? [Send a bug report](https://github.com/algolia/instantsearch.js/issues/new?template=Bug_report.md).
-* **Feature request**. Would you like to add something to the library? [Send a feature request](https://github.com/algolia/instantsearch.js/issues/new?template=Feature_request.md).
-* **Documentation**. Did you find a typo in the doc? Click on the `edit this page` button.
```

Comment:
This is not the case anymore since we migrated the documentation website to Algolia.